src/DIRAC/TransformationSystem/Client/TransformationClient.py

```diff
@@ -173,34 +174,37 @@ def getTransformationFiles(
             condDict = {}
         if timeStamp is None:
             timeStamp = "LastUpdate"
-        # getting transformationFiles - incrementally
-        if "LFN" in condDict:
-            if isinstance(condDict["LFN"], str):
-                lfnList = [condDict["LFN"]]
-            else:
-                lfnList = sorted(condDict["LFN"])
-            # If a list of LFNs is given, use chunks of 1000 only
-            limit = limit if limit else 1000
+
+        if "LFN" not in condDict:
+            res = rpcClient.getTransformationFiles(
+                condDict, older, newer, timeStamp, orderAttribute, offset, maxfiles, columns
+            )
+            if not res["OK"]:
+                return res
+            return S_OK(res["Value"])
```

Comment:
```suggestion
            return res
```
Or is there something from res we want to filter out when `OK`? (Which would then deserve a comment)